tests/common/storages/utils.py

```diff
@@ -49,7 +49,8 @@ def assert_sample_files(
 
     for item in all_file_items:
         # only accept file items we know
-        assert item["file_name"] in minimally_expected_file_items
+        if not item["file_name"] in minimally_expected_file_items:
+            continue
```

Comment:
That seems like a very brutal check. In the Google Drive case it doesn't work, because there is another folder in the bucket (in addition to the `samples`).